src/templates/ObservabilityPackDetails.js

```diff
@@ -24,9 +25,22 @@ const ObservabilityPackDetails = ({ data, location }) => {
               gap: 1rem;
             `}
           >
-            <Button variant={Button.VARIANT.PRIMARY} size={Button.SIZE.MEDIUM}>
-              Install
-            </Button>
+            <InstallButton
+              title="Install"
```

Comment:
We might want to consider adding the pack name to the title:

```suggestion
              title={`Install ${pack.name}`}
```

Reply:
I think that's a good call, but wasn't in the original design. Should we go for it? It's simples to change back should we need to?